pom.xml

```diff
@@ -839,18 +839,29 @@
             <instructions>
               <Bundle-SymbolicName>${bundle-symbolic-name}</Bundle-SymbolicName>
               <Bundle-Description>Jetty module for ${project.name}</Bundle-Description>
-              <Bundle-RequiredExecutionEnvironment>JavaSE-11</Bundle-RequiredExecutionEnvironment>
               <Bundle-DocURL>${jetty.url}</Bundle-DocURL>
               <Bundle-Vendor>Eclipse Jetty Project</Bundle-Vendor>
               <Bundle-Classpath>.</Bundle-Classpath>
               <Bundle-Copyright>Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.</Bundle-Copyright>
               <Import-Package>
                 ${osgi.slf4j.import.packages},
+                !java.io,
+                !java.lang,
+                !java.lang.annotation,
+                !java.lang.invoke,
+                !java.lang.reflect,
+                !java.net,
+                !java.nio.charset,
+                !java.nio.file,
+                !java.util,
+                !java.util.function,
+                !java.util.stream,
+                !javax.xml.catalog,
+                !javax.xml.parsers,
```

Comment:
In case you just want to remove all java.* packages you can simply use `!java.*,`.
Alternatively you could use [`<_noimportjava/>`](https://bnd.bndtools.org/instructions/noimportjava.html)
But as said excluding those packages should not be necessary.

Reply:
Removed.  

I didn't add `<_noimportjava/>`.

Reply:
Perfect. Thanks!